README.md

```diff
@@ -62,20 +63,21 @@ This will run through the assets precompilation, linting and testing before
 creating a Docker image, then you can run the image with the following command:
 
 ```sh
-APPLICATION_ROOT=/ make run
+make run
 ```
 
-N.B In the production environment, the app will be accessed via the URL path `/app/root`.
-When running the docker image locally for development, you may need to access the
-application via a proxy, otherwise, the paths for JS, CSS and image assets might
-not work.
+N.B In the production environment, the app will be accessed via the URL path
+`/app/root`. When running the docker image locally for development, you may need
+to access the application via a proxy, otherwise, the paths for JS, CSS and
+image assets might not work.
 
-Please see the *[simple web proxy](https://github.com/epimorphics/simple-web-proxy)*
-repository for one simple way of handling this on a local develpment machine.
+Please see the *[simple web
+proxy](https://github.com/epimorphics/simple-web-proxy)* repository for one
+simple way of handling this on a local develpment machine.
 
 With the proxy and Docker container running you can access the application as
-[`localhost:3001/app/root/`](http://localhost:3001/app/root/)
-(note the trailing path).
+[`localhost:3001/app/root/`](http://localhost:3001/app/root/) (note the trailing
```

Comment:
This is inconsistent with ports in production.
Move proxy from 3001.

Reply:
I agree, I used `localhost:3030` in my changes which I will push up in a new PR for the simple web proxy repository

Reply:
I've reconsidered this. I think the proxy should listen on 8080.

Reply:
The issue with using `8080` is that's the API port: `http://localhost:8080` set in the `API_SERVICE_URL` default value? 

Mind you I only went with `3030` as it was still in the 3k range but we could choose something that works for all the different endpoints?